Word-to-PDF-Conversion/Replace-Embedded-Excel-as-Image/Replace-Embedded-Excel-as-Image/Program.cs

```diff
@@ -24,13 +21,9 @@
 FileStream outputStream = new FileStream(Path.GetFullPath(@"Output/Output.pdf"), FileMode.Create, FileAccess.Write);
 // Save the generated PDF to the specified file stream
 pdf.Save(outputStream);
```

Comment:
Please remove the setting options
docIORenderer.Settings = new DocIORendererSettings();

And also use "using" statement for File stream open and save